news/2021-09-18-200th-featured-artist-our-stolen-theory.md

```diff
@@ -0,0 +1,67 @@
+
+---
+
+As new osu! players climb ranks and older ones retire, some major events in the community fade from memory.
+
+Even so, [Azer](https://osu.ppy.sh/users/2155578)'s journey to full combo-ing [the first ranked map of *United (L.A.O.S Remix)*](https://osu.ppy.sh/beatmapsets/237768) with HDHR is an event that'll never be forgotten.
+
+Regardless of whether you're a new player or an osu! veteran, relive the excitement of this play through Azer's video, then play through any of the other awesome beatmaps of listed below!
+
+<iframe width="100%" height="400" src="https://www.youtube.com/embed/VkjtPz47UQs" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+### Our Stolen Theory - United (L.A.O.S Remix)
+
+Experience this timeless classic for yourself through any of these top-tier beatmaps:
+
+- [osu! map](https://osu.ppy.sh/beatmapsets/237768) by [Asphyxia](https://osu.ppy.sh/users/1715720) - the first ranked and most popular of the bunch!
+- [osu! map](https://osu.ppy.sh/beatmapsets/953303) by [Sotarks](https://osu.ppy.sh/users/4452992) - a modern rendition showcasing a different style!
+- [osu!taiko map](https://osu.ppy.sh/beatmapsets/559371) by [hikiko-](https://osu.ppy.sh/users/6512678)
+- [another osu!taiko map](https://osu.ppy.sh/beatmapsets/556731) by [Yuzeyun](https://osu.ppy.sh/users/481582)
+- [osu!catch map](https://osu.ppy.sh/beatmapsets/527777) - loved beatmap featuring an outstanding storyboard!
+- [osu!mania map](https://osu.ppy.sh/beatmapsets/726545) - a full spread for 4K players!
+
+[Download the beatmap template](https://assets.ppy.sh/artists/200/Songs/Our%20Stolen%20Theory%20-%20United%20(L.A.O.S%20Remix).osz) to make your own map of this song!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/200/Songs/Our%20Stolen%20Theory%20-%20United%20(L.A.O.S%20Remix).mp3" type="audio/mpeg">
+</audio>
+
+---
+
+[**Our Stolen Theory**](https://osu.ppy.sh/beatmaps/artists/200) is osu!'s 200th Featured Artist. That's a big deal.
+
+It's honestly hard to believe how quickly things have blown up around here. We reached [**100 artists**](https://osu.ppy.sh/home/news/2020-09-23-new-featured-artist-epica) less than a year ago, and it took **4 years** since the start of the Featured Artist program to reach that milestone.
+
+Raw numbers might help put the current size of osu!'s Featured Artist library into perspective:
+
+- **4,326 tracks from 200 artists** are available for mapping from our Featured Artists!
+- **2,320 beatmaps have been ranked** using Featured Artist songs!
+- Meanwhile, approximately **33,701 ranked/pending/work-in-progress beatmaps** use Featured Artist songs! That's more than half of all non-graveyard maps.
```

Comment:
since it's usually rank**ed**, lov**ed**, approv**ed** and qualifi**ed** ~~and pending**ed**~~ beatmaps

```suggestion
- Meanwhile, approximately **33,701 ranked/pending/work-in-progress beatmaps** use Featured Artist songs! That's more than half of all non-graveyarded maps.
```